src/c++/vdo/base/io-submitter.h

```diff
@@ -24,24 +24,23 @@ void process_vio_io(struct vdo_completion *completion);
 
 void submit_data_vio_io(struct data_vio *data_vio);
 
-void vdo_submit_metadata_io(struct vio *vio, physical_block_number_t physical,
-			    bio_end_io_t callback, vdo_action *error_handler,
-			    unsigned int operation, char *data);
+void __submit_metadata_vio(struct vio *vio, physical_block_number_t physical,
+			   bio_end_io_t callback, vdo_action *error_handler,
+			   unsigned int operation, char *data);
 
-static inline void submit_metadata_vio(struct vio *vio, physical_block_number_t physical,
-				       bio_end_io_t callback, vdo_action *error_handler,
-				       unsigned int operation)
+static inline void vdo_submit_metadata_vio(struct vio *vio, physical_block_number_t physical,
+					   bio_end_io_t callback, vdo_action *error_handler,
+					   unsigned int operation)
 {
-	vdo_submit_metadata_io(vio, physical, callback, error_handler, operation,
-			       vio->data);
+	__submit_metadata_vio(vio, physical, callback, error_handler, operation, vio->data);
 }
```

Comment:
I think I decided to wrap this line even though Mike didn't. Be mindful of that when rebasing.